Add dev package to docs (#2578)
DataDog/integrations-core: docs/datadog_checks_dev.rst

.. _datadog_checks_dev:

datadog_checks_dev
==================

.. toctree::
    :maxdepth: 3

    datadog_checks_dev.api
    datadog_checks_dev.cli

Installation
------------

.. code-block:: bash

    $ pip install "datadog-checks-dev[cli]"

> At this point there should be a working executable, :ref:`ddev`, in your PATH.

Reviewer: ```suggestion
This results in a working executable, :ref:`ddev`, in your PATH.
```

If you make this a conditional ("should"), then you also need to explain other modes (i.e. the executable is _not_ there). For now, recommend using a declaration instead. :)
